stix_shifter_modules/arcsight/stix_transmission/api_client.py

```diff
@@ -0,0 +1,116 @@
+import time
+from datetime import datetime, timedelta
+import json
+from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
+from stix_shifter_utils.utils.error_response import ErrorResponder
+
+
+class APIClient:
+    USER_TOKEN = 'core-service/rest/LoginService/login'
```

Comment:
rename this to `TOKEN_ENDPOINT`

Reply:
The proposed change is made and updated to the PR branch.